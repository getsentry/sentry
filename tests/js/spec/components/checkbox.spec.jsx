import React from 'react';

import {mountWithTheme} from 'sentry-test/enzyme';

import Checkbox from 'app/components/checkbox';

describe('Checkbox', function() {
  it('renders', function() {
    const component = mountWithTheme(<Checkbox onChange={() => {}} />);

    expect(component).toSnapshot();
    expect(component).toMatchSnapshot();
  });
});
