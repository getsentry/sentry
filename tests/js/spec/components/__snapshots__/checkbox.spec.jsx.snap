// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders 1`] = `
<input
  checked={false}
  type="checkbox"
/>
`;
